### DISCUSSION TITLE
UI and popups when pressing buttons

### DISCUSSION THREAD

**Discussion by nasawyer7** [seerge/g-helper#988] [General]
This tool is amazing. Love it. But, there are some issues when pressing hotkeys. So, the mute hotkey only has a popup on my screen once asus services are stopped. Before then, the mute hotkey works, but does not tell me if my computer is muted or not, which can get confusing. 

Also, I was hoping for the aura key to have a popup to tell you what aura mode you are on, to make it less confusing to switch between them without opening the ui. Without opening the ui, its very difficult to actually make use of the aura button. 

Thank you for reading!

**Comment by seerge** [comment]
@nasawyer7 when asus services are running g-helper is not handling your hotkeys at all, so it's up to asus what to show :) 

Aura key doesn't show a notification cause there is too many variants on some models + you kind of see result immediately on keyboard itself

**Reply by nasawyer7** [reply]
ohh ok well i just manually deleted all running asus services by claiming permissions 

alright! maybe some people might want this feature later idk thanks for the fast response though!
